File: template.c
#include <stdio.h>

#ifndef BIGBOY
//
#define INPUT "input.txt"
#define SOLUTION (-1)

#else
//
#define INPUT "bigboy.txt"
#define SOLUTION (-1)
#endif

int main()
{
    FILE *file = fopen(INPUT, "r");

    char c = fgetc(file);

    fclose(file);

    int success = solution == SOLUTION;
    printf("Solution: %d (%d)\n", solution, success);

    return (success) ? 0 : 1;
}

File: template.py
import os
from typing import TypeVar, Union

T = TypeVar("T")
OptT = Union[T, None]

SOLUTION = -1


def main() -> int:
    input_path = os.path.join(os.path.split(__file__)[0], "input.txt")

    fd = open(input_path, "r")

    l = fd.readline().strip()
    while len(l) != 0:
        l = fd.readline().strip()

    fd.close()

    success = solution == SOLUTION
    print(f"Solution: {solution} ({success})")

    return 0 if success else 1


if __name__ == "__main__":
    exit(main())
